supplementary/constraintsolver/test/test_cnsmtgsolver.cpp

```diff
@@ -19,12 +19,12 @@ using namespace alica::reasoner;
 using namespace alica::reasoner::cnsat;
 using namespace autodiff;
 
-const int minCount = 2, maxCount = 20;
-const int minform = 2, maxform = 2;
+// const int minCount = 2, maxCount = 20;
```

Comment:
Remove?

Reply:
this variables belongs to commented below code

Reply:
In general i'd say remove all the commented out code, it's not clear why it's there, so it's a dead weight. But maybe it's actually needed.